### PR TITLE
MicrosoftCorporation.MicrosoftReportBuilder version 15.0.19914.0

### LINKED ISSUE #75355: [Package Request]: Microsoft Report Builder
### Package Requested

- [X] I would like someone else to build the manifest.
- [X] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Microsoft
Publisher Url: https://www.microsoft.com/en-us/download/details.aspx?id=53613
Package Name: Microsoft Report Builder
Description: Report Builder provides a productive report-authoring environment for IT professionals and power users. The download provides a stand-alone installer for Report Builder.

Report Builder provides data visualizations that include charts, maps, sparklines, and data bars that can help produce new insights well beyond what can be achieved with standard tables and charts. Use Report Builder to create reports and shared datasets. Publish report parts, and then browse the Report Part Gallery to reuse existing report parts as building blocks for creating new reports quickly with a “grab and go” experience.

Package Version: 15.0.19914.0
Installer URL: https://download.microsoft.com/download/5/E/B/5EB40744-DC0A-47C0-8B0A-1830E74D3C23/ReportBuilder.msi

Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by randomgit87**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/75356)

### CODE REVIEW

File: manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftReportBuilder
```
Comment (Trenly) [inline manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.ReportBuilder
```
A folder already exists for Microsoft as a publisher, so the simpler Identifier is better

File: manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftReportBuilder
+PackageVersion: 15.0.19914.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft Report Builder
+License: Licensed
```
Comment (Trenly) [inline manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftReportBuilder
```
Comment (Trenly) [inline manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.ReportBuilder
```
A folder already exists for Microsoft as a publisher, so the simpler Identifier is better

File: manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftReportBuilder
```
Comment (Trenly) [inline manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.ReportBuilder
```
A folder already exists for Microsoft as a publisher, so the simpler Identifier is better

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/75356#pullrequestreview-1088805541

**Review by randomgit87 — COMMENTED** [review]
comment for aproveal

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
1) You need to change the `MicrosoftCorporation` folder to `Microsoft`
2) You need to change the `MicrosoftReportBuilder` folder to `ReportBuilder`
3) You also need to do the same for the file names

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/75356#pullrequestreview-1095503447

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 79217;

**Pull request closed without merging**